assets/users/components/EditUser.tsx

```diff
@@ -314,26 +334,6 @@ function EditUserComponent({
     );
 }
 
-EditUserComponent.propTypes = {
```

Comment:
move info from propTypes to `IProps`

Reply:
I have already moved that info to IProps